### ISSUE TITLE
Crash when pressing on the last loaded multireddit submission

### ISSUE THREAD

**Issue by Deadleg** [ccrama/Slide#1034]
Slide version: 1da95b632517092d670fb03b5c2ad7a612b8b2c6
Android version: 5.0

Scrolling to the last post in loaded in a multireddit crashes with the below stacktrace. Note this seems to only happen on multireddits, not subreddits.

```
FATAL EXCEPTION: main
                                                                       Process: me.ccrama.redditslide, PID: 28857
                                                                       java.lang.IndexOutOfBoundsException: Inconsistency detected. Invalid view holder adapter positionViewHolder{266ffbfb position=301 id=-1, oldPos=101, pLpos:101 scrap [attachedScrap] tmpDetached no parent}
                                                                           at android.support.v7.widget.RecyclerView$Recycler.validateViewHolderForOffsetPosition(RecyclerView.java:4251)
                                                                           at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4382)
                                                                           at android.support.v7.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:4363)
                                                                           at android.support.v7.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:1961)
                                                                           at android.support.v7.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1370)
                                                                           at android.support.v7.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1333)
                                                                           at android.support.v7.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:562)
                                                                           at android.support.v7.widget.RecyclerView.dispatchLayout(RecyclerView.java:2864)
                                                                           at android.support.v7.widget.RecyclerView.onLayout(RecyclerView.java:3071)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.support.v4.widget.SwipeRefreshLayout.onLayout(SwipeRefreshLayout.java:584)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1076)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.support.v4.view.ViewPager.onLayout(ViewPager.java:1627)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.RelativeLayout.onLayout(RelativeLayout.java:1076)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.FrameLayout.layoutChildren(FrameLayout.java:573)
                                                                           at android.widget.FrameLayout.onLayout(FrameLayout.java:508)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.FrameLayout.layoutChildren(FrameLayout.java:573)
                                                                           at android.widget.FrameLayout.onLayout(FrameLayout.java:508)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1705)
                                                                           at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1559)
                                                                           at android.widget.LinearLayout.onLayout(LinearLayout.java:1468)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.FrameLayout.layoutChildren(FrameLayout.java:573)
                                                                           at android.widget.FrameLayout.onLayout(FrameLayout.java:508)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1705)
                                                                           at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1559)
                                                                           at android.widget.LinearLayout.onLayout(LinearLayout.java:1468)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at me.ccrama.redditslide.SwipeLayout.SwipeBackLayout.onLayout(SwipeBackLayout.java:389)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.widget.FrameLayout.layoutChildren(FrameLayout.java:573)
                                                                           at android.widget.FrameLayout.onLayout(FrameLayout.java:508)
                                                                           at android.view.View.layout(View.java:15631)
                                                                           at android.view.ViewGroup.layout(ViewGroup.java:4966)
                                                                           at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2101)
                                                                           at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:1858)
                                                                           at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1077)
                                                                           at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:5845)
                                                                           at android.view.Choreographer$CallbackRecord.run(Choreographer.java:767)
                                                                           at android.view.Choreographer.doCallbacks(Choreographer.java:580)
                                                                       	at and
```

**Comment by Deadleg** [comment]
Fixed with 4a1cd7940afd2866a28942fb4d89ff13bb4db3a7.

**Comment by ccrama** [comment]
Glad it's fixed! Also would it be possible for you to get on IRC for a minute?
